src/libs/core.py

```diff
@@ -7,7 +7,7 @@
 import flask
 import tweepy
 
-CONFIG_PATH = 'config/twitter_account_manager.ini'
+CONFIG_PATH = 'config/config.cfg'
```

Comment:
I agree with changing the config file name. However, you need to update the example file as well as .gitignore file. That way, we can ensure that people are not pushing their consumer keys ;-)

Reply:
;)